en/config/actuators.md

```diff
@@ -313,6 +313,51 @@ Configuring the Camera Capture / Trigger input requires a reboot to take effect
 You should assign functions to the outputs that match your physical wiring of motors and servos, and use the [Actuator Testing](#actuator-testing) section described below to determine appropriate output parameter values.
 These steps are covered in [Output Assignment and Configuration](#output-assignment-and-configuration).
 
+### Output Functions
+
+Output functions are used to map the "logical functions" of an airframe, such as `Motor 1` or `Landing gear`, to physical outputs like FMU output pin 2.
+This makes it easy to use a particular output pin for almost any purpose.
+
+Some functions are only relevant to particular frames or output types, and will not be offered on others.
+
+Functions include: 
+
+- `Disabled`: Output has no assigned function.
+- `Constant_Min`: Output set to constant minimum value.
+- `Constant_Max`: Output is set to constant maximum value.
+- `Motor 1` to `Motor 12`: Output is indicated motor.
+  Only motors allowed for airframe are displayed.
+- `Servo 1` to `Servo 8`: Servo output.
+   These are further assigned a specific meaning based on airframe, such as "tilt servo", "left aileron".
+- `Offboard Acutator Set 1` to `Offboard Acutator Set 6`: MAVLink Payload output.
+- `Landing Gear`: Output is landing gear.
+- `Parachute`: Output is parachute.
+- `RC Roll`: Output is roll from RC.
+- `RC Pitch`: Output is pitch from RC.
+- `RC Throttle`: Output is throttle from RC.
+- `RC Yaw`: Output is yaw from RC
+- `RC Flaps`: Output is flaps from RC
```

Comment:
These are just RC direct passthrough right? So these can be assigned via other params to particular channels?

Reply:
Yes this is the direct passthrouh. Which 'other params' are you referring to?

Reply:
https://docs.px4.io/main/en/advanced_config/parameter_reference.html#RC_MAP_PITCH and friends. These say what channel on the is passed through the output identified by `RC Pitch`and friends. I'll update that.